.circleci/config.yml

```diff
@@ -161,8 +161,6 @@ workflows:
               only: /.*/
       - build_and_push:
           context: org-global
-          requires:
```

Comment:
Was that intentional to remove the test?

Reply:
Yeah I don't think it's necessary.

Reply:
tests still run--they just don't block building the docker image